Makefile

```diff
@@ -982,6 +982,12 @@ test-test-extension-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integra
 	$(GOTESTSUM) --junitfile $(ARTIFACTS)/junit.test_extension.xml --raw-command cat $(ARTIFACTS)/junit.test_extension.stdout
 	exit $$(cat $(ARTIFACTS)/junit.test_extension.exitcode)
 
+.PHONY: test-framework-junit
+test-framework-junit: $(SETUP_ENVTEST) $(GOTESTSUM) ## Run unit and integration tests and generate a junit report for the test extension
```

Comment:
Is this related to the KAL SSATags?

Reply:
My bad. Should not be part of this PR

Reply:
Rebased on top of main, should be gone now